frontend/renderer.js

```diff
@@ -1,117 +1,76 @@
 import * as THREE from '../lib/three.module.js';
 import { OrbitControls } from '../lib/OrbitControls.js';
-import { CSS2DRenderer, CSS2DObject } from '../lib/CSS2DRenderer.js';
 
 import * as Rad from '../radiosity/index.js';
 import * as Modeling from '../modeling/index.js';
 
-import delays from './tools/delays.js';
-import setupMenu from './menu.js';
+import * as axes from './tools/axes.js';
+import * as components from './tools/basic-components.js';
 
-// list of available environments; the first one is the default
+export const viewParameters = {
+  viewOutput: new components.Toggle('view radiosity output', false), // the current view is either vertex (radiosity) or shaded
+  viewWireframe: new components.Toggle('wireframe view', false),
```

Comment:
I think it's better to have the parameters declared this way.
It makes the renderer.js file finally readable.